x/time/time.go

```diff
@@ -0,0 +1,17 @@
+package time
+
+import "time"
+
+// UnixNano is used to indicate that an int64 stores a unix timestamp at
+// nanosecond resolution
+type UnixNano int64
+
+// ToTime returns a time.ToTime from a UnixNano
+func (u UnixNano) ToTime() time.Time {
```

Comment:
Might be worth putting this into `m3x/time` and using package name of `xtime` so its more similar to our other shared types.  At some point `m3aggregator` will probably want to use the same approach and they may as well reference the same type.